charts/vector/README.md

```diff
@@ -63,6 +63,25 @@ helm upgrade -f values.yaml <RELEASE_NAME> vector/vector
 
 See the [All configuration options](#all-configuration-options) section to discover all possibilities offered by the Vector chart.
 
+### Using template syntax in `customConfig`
+
+As Vector's [template syntax](https://vector.dev/docs/reference/configuration/template-syntax/) shares the same syntax as Go templates, explicit handling is required
```

Comment:
```suggestion
As Vector's [template syntax](https://vector.dev/docs/reference/configuration/template-syntax/) shares the same syntax as Helm templates, explicit handling is required
```

Users may not realize that Helm is using Go's templating language since they don't seem to emphasize than in their docs.

Reply:
🤔 happy to change it but: https://helm.sh/docs/howto/charts_tips_and_tricks/#know-your-template-functions

Reply:
True, but that doc is aimed at chart developers rather than chart users.